<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adding_exercise</class>
 <widget class="QWidget" name="adding_exercise">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dodawanie ćwiczenia</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/add_task_black_24dp.svg</normaloff>:/icons/add_task_black_24dp.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,1">
     <item>
      <widget class="QLabel" name="Nazwa_cwiczenia_L">
       <property name="text">
        <string>Nazwa ćwiczenia.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="Name_CB"/>
     </item>
     <item>
      <widget class="QLabel" name="Opis_cwiczenia_L">
       <property name="text">
        <string>Opis ćwiczenia.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="Description_TB"/>
     </item>
     <item>
      <widget class="QLabel" name="GrupaC_L">
       <property name="text">
        <string>Grupa ćwiczenia</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="GroupEx_CB">
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="RepsL">
         <property name="text">
          <string>Powtórzenia</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="RepsS"/>
       </item>
       <item>
        <widget class="QLabel" name="WeightL">
         <property name="text">
          <string>Waga</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="WeightDS">
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.250000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,4,1,1">
       <item>
        <widget class="QPushButton" name="addNewExB">
         <property name="text">
          <string>Nowe ćwiczenie</string>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="AcceptB">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/add_task_black_24dp.svg</normaloff>:/icons/add_task_black_24dp.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DeclineB">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/highlight_off_black_24dp.svg</normaloff>:/icons/highlight_off_black_24dp.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
